Cargo.toml

```diff
@@ -167,3 +167,7 @@ debug = 0
 inherits  = "release"
 opt-level = 1
 lto       = "off"
+
+[profile.release]
+# https://doc.rust-lang.org/cargo/reference/profiles.html#strip
+strip = true
```

Comment:
This step should be in this case deleted, right? https://github.com/ChainSafe/forest/blob/main/.github/workflows/release.yml#L42

Reply:
Good catch! fixed